Code/Tools/AssetProcessor/native/tests/PathDependencyManagerTests.cpp

```diff
@@ -338,8 +338,7 @@ namespace UnitTests
     }
 
     struct PathDependencyBenchmarks
-        : ::UnitTest::ScopedAllocatorFixture
-          , PathDependencyBase
+        : PathDependencyBase
     {
```

Comment:
This change removes the allocator fixture parent class, is it intentional?

Reply:
It was conflicting with the base class of the `PathDependencyTestValidation` fixture. But this change was wrong anyway, I've added it back. and removed it from the more-derived type `PathDependencyTestValidation`.